src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private static List<Task> taskList;
```

Comment:
I feel some might misinterpret `taskList` to be an instance of the TaskList class, perhaps consider `tasks`?

Reply:
yup i think that will make it more clear!